libraries/aws/shadow/src/aws_iot_shadow_operation.c

```diff
@@ -919,7 +909,7 @@ AwsIotShadowError_t _AwsIotShadow_ProcessOperation( IotMqttConnection_t mqttConn
         status = AWS_IOT_SHADOW_STATUS_PENDING;
```

Comment:
Isn't this the value with which `status` variable was initialized?

Reply:
Yeah, but in the middle it was converted to "SUCCESS" in happy path.